add merge date notice (#7253)
ethereum/ethereum-org-website: src/content/upgrades/merge/index.md

For more information, check out this blog post by Tim Beiko on [How The Merge Impacts Ethereum’s Application Layer](https://blog.ethereum.org/2021/11/29/how-the-merge-impacts-app-layer/).
</ExpandableCard>

> ## When will The Merge happen? {#wen-merge}

Reviewer: ```suggestion
## What date is The Merge? {#wen-merge}
```

@samajammin I think adding 'merge date' to the heading makes a bit more sense here.

> ## When will The Merge happen? {#when-merge}

Reviewer: 😜

```suggestion
## When will The Merge happen? {#wen-merge}
```

> The Merge is expected to happen in Q3/Q4 2022. The client developers are currently working to a soft deadline of 19th September 2022, but this could change depending upon the success of the final testnet merge (Goerli) in mid-August, continued client refinements and the hashrate of the existing miners continuing in a predictable manner. Everyone is working hard to deliver The Merge as soon as possible.

Reviewer 2: ```suggestion
The Merge date is expected to land within Q3/Q4 2022. The client developers are currently working to a soft deadline of 19th September 2022, but this could change depending upon the success of the final testnet merge (Goerli) in mid-August, continued client refinements and the hashrate of the existing miners continuing in a predictable manner. Everyone is working hard to deliver The Merge as soon as possible.
```
Not sure if this is a great way to phrase it but it's important to have "merge date" as a keyword somewhere in here 😄

> The Merge date is expected to land within Q3/Q4 2022. The client developers are currently working to a soft deadline of 19th September 2022, but this could change depending upon the success of the final testnet merge (Goerli) in mid-August, continued client refinements and the hashrate of the existing miners continuing in a predictable manner. Everyone is working hard to deliver The Merge as soon as possible.

Reviewer: ```suggestion
The Merge is expected to land within Q3/Q4 2022. The client developers are currently working to a soft deadline of 19th September 2022, but this could change depending upon the success of the final testnet merge (Goerli) in mid-August, continued client refinements and the hashrate of the existing miners continuing in a predictable manner. Everyone is working hard to deliver The Merge as soon as possible.
```
